packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -148,157 +107,193 @@
+                        data-ng-if="$row.isVirtual()"
+                    >
+                        <span
+                            data-translate="dedicated_server_interfaces_rename_label"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        data-ui-sref="app.dedicated-server.server.interfaces.bandwidth-public-order"
+                        data-ng-if="$row.isPublic() && $ctrl.server.canOrderBandwith"
+                    >
+                        <span
+                            data-translate="dedicated_server_interfaces_order_public_bandwidth_label"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        data-ui-sref="app.dedicated-server.server.interfaces.bandwidth-private-order"
+                        data-ng-if="$row.isPrivate() && $ctrl.server.canOrderVrackBandwith"
+                    >
+                        <span
+                            data-translate="dedicated_server_interfaces_order_private_bandwidth_label"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        data-ui-sref="app.ip({ serviceName: $ctrl.serverName })"
+                        data-ng-if="$row.isPublic()"
+                        data-external
+                    >
+                        <span
+                            data-translate="dedicated_server_interfaces_manage_ips_label"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        data-ui-sref="app.dedicated-server.server.interfaces.attach({interface: $row})"
+                        data-ng-if="!$row.hasTaskInProgress() && $row.isPrivate() && !$row.hasVrack()"
+                    >
+                        <span
+                            data-translate="dedicated_server_interfaces_attach_private_network_label"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        data-ui-sref="app.dedicated-server.server.interfaces.detach({interface: $row})"
+                        data-ng-if="!$row.hasTaskInProgress() && $row.hasVrack()"
+                    >
+                        <span
+                            data-translate="dedicated_server_interfaces_detach_private_network_label"
+                        ></span>
+                    </oui-action-menu-item>
+                </oui-action-menu>
+            </oui-datagrid>
+        </oui-tile>
+    </div>
+</div>
+
+<!-- QUOTA -->
+<div class="mb-5" data-ng-if="$ctrl.trafficInformations.traffic.data.hasQuota"
+    data-ng-include="'dedicated/server/consumption/consumption.html'">
+</div>
+
+<!-- MRTG -->
```

Comment:
FYI here I have removed the `data-ng-if="!$ctrl.server.isExpired && $ctrl.displayMRTG()"` condition that was here before
This is because I didn't manage to get this working.
But I'm not sure whether it's necessary or not.
- `!$ctrl.server.isExpired`: For expired server I could not reproduce it, but if it's expired, does it matter to have the (translated) error for chart not loading ? I mean the guy didn't pay so...
- `$ctrl.displayMRTG()`: the function basically exclude the following server ranges: `HIDE_MRTG_FOR_SERVER_RANGES = ['scale', 'hgr']`
HGR: displaying the chart works, just translation is missing...

Reply:
No after talking with baremetal-system network team, I have decided to re-established it